Add python client docs to docs/, run in CI (#1086)
G-Research/armada: docs/libraries.md

# Libraries 

## Overview

> Client libraries for Armada are under active development. This document is intended to document what clients exist and how to use them.

Reviewer: ```suggestion
This document is an overview of Armada client libraries.
```

## Python

> Armada provides python client bindings providing access to the RPC API. 

Reviewer: ```suggestion
Here, we give an overview of the Armada Python client. 
```
